R/numerical_eda.R

```diff
@@ -0,0 +1,59 @@
+#' Numerical Plotting Function
+#'
+#' This function takes in a tibble or dataframe object, two numeric columns,
+#' and produces either a scatter or line plot to visualize the relationship
+#' between the two numerical features. Users can optionally change default
+#' arguments for plot-type, color, title, size of text, color-scheme, and
+#' toggle log transformation for the x and y axis.
+#'
+#' @param data A tibble or data frame object.
+#' @param xval A character variable used to represent the x-axis.
+#' @param yval A  variable used to represent the y-axis.
+#' @param plot_type An optional character variable used to represent
+#' the graphical relationship between xval and yval,
+#' options are scatter or line plot.
+#' @param color An optional character variable used to group the data
+#' points in different colors.
+#' @param title An optional character variable used to set the title of the plot.
+#' @param font_size An optional integer variable used to set the font size.
```

Comment:
specify font size for the plot title and axis.

Reply:
I have updated it